github_pr.py

```diff
@@ -369,8 +395,16 @@ def main():
 Delete a PR
 
     github-pr delete -r dataxu/test_repo -n 17
+
+Check conditional status checks
+
+    github-pr check-condition -r dataxu/dcommand -n 84 --condition-owner-cannot-ship
+        This will check to make sure that the owner can not apply a shipable comment on their own code
+        Also available at this time is a check to make the comment is from an approved shipper
+        This check only looks at comments AFTER the latest commit, to validate that the
+        code has been most recently peer reviewed.
```

Comment:
Do we want to change the conditional name @dx-pbuckley? I thought we had said that we thought that the condition should also be like the "owner-cannot-ship" context? 

maybe something like "--condition-found-approved-merger" or "--condition-approved-merger-comment-found"

Also i noticed you added the "--condition-approved-mergers" as an option here. Seeing as this would prob not be used other than programmatic i decided against that, but I can add it as the other status checks, assuming it would be returning the same status at the "approved-mergers-file"